com.unity.render-pipelines.core/Editor/AdditionalPropertiesHandler.cs

```diff
@@ -0,0 +1,14 @@
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// Interface for editors using Additional Properties with EditorPrefBoolFlags
+    /// </summary>
+    public interface IAdditionalPropertiesBoolFlagsHandler
```

Comment:
I did that because we also have VolumeComponentEditors which are not using BoolFlag but just a BoolPref and are not handled by this one so I wanted to make it obvious.

Reply:
Oh yes right. You are good then.